src/ageAssurance/components/NoAccessScreen.tsx

```diff
@@ -182,6 +196,17 @@ export function NoAccessScreen() {
                   <Trans>Add your birthdate</Trans>
                 </ButtonText>
               </Button>
+
+              {isUsingAppPassword && (
+                <Admonition type="info">
+                  <Trans>
+                    Hmmm, looks like you're logged in with an{' '}
```

Comment:
```suggestion
                    Hmm, it looks like you're logged in with an{' '}
```
nit: minor copy tweak suggestion

(two `m`s in `Hmm` is major-league pedantry I know, but that's the most-common usage in other strings fwiw)